### ISSUE TITLE
Electrum crashed when i click exit

### ISSUE THREAD

**Issue by ElleryHao** [spesmilo/electrum#6653]
I use the v4.0.4 electrum of windows, when i closed the electrum, then crash infomations came up

**Comment by ecdsa** [comment]
what kind of crash? what information?

**Comment by verretor** [comment]
Maybe related. https://github.com/spesmilo/electrum/issues/3640

**Comment by ElleryHao** [comment]
> what kind of crash? what information?
there is no enough infomation, just a crash alarm info came up

**Comment by ecdsa** [comment]
closing, not useful

**Comment by HusnaQA** [comment]
> I use the v4.0.4 electrum of windows, when i closed the electrum, then crash infomations came up

I experienced something similar on macOS 10.15.7. When I closed the Electrum 4.0.4 application, I had to Quit the application (⌘Q) several times.
![Screen Shot 2020-11-11 at 21 25 31](https://user-images.githubusercontent.com/51465672/98875743-665a4200-24af-11eb-9ad1-1b6463c70bb9.png)

I also tried to use the Ledger Nano X with Electrum 4.0.4 and experienced the same thing when I closed the application; it didn't want to quit. I tried to open existing menus, Electrum preferences; it remained open even though the Electrum main display was blank.
![Screen Shot 2020-11-12 at 05 37 14](https://user-images.githubusercontent.com/51465672/98875762-7114d700-24af-11eb-8e23-870ba4dfad78.png)
